Makefile

```diff
@@ -0,0 +1,15 @@
+build :
+	docker-compose build
+
+clean :
+	docker-compose down --remove-orphans
+	docker image prune
+	docker volume prune
+
+services :
+	docker-compose build --parallel es memcached redis pg
+	docker-compose up -d es memcached redis pg
+	docker-compose -f ../iNaturalistAPI/docker-compose.yml -f ../iNaturalistAPI/docker-compose.override.yml up --build
```

Comment:
This is the main part I'm concerned with and still treating as WIP as it makes assumptions on file structures. Curious what your thoughts are. Maybe there's a workaround, or maybe documentation around container caveats/requirements.